homeassistant/components/flux_led/config_flow.py

```diff
@@ -0,0 +1,220 @@
+                            CONF_HOST: bulb["ipaddr"],
+                        }
+
+            return self.async_create_entry(
+                title="FluxLED/MagicHome",
+                data={
+                    CONF_AUTOMATIC_ADD: user_input[CONF_AUTOMATIC_ADD],
+                    CONF_EFFECT_SPEED: DEFAULT_EFFECT_SPEED,
+                    CONF_DEVICES: devices,
+                },
+            )
+
+        return self.async_show_form(
+            step_id="user",
+            data_schema=vol.Schema(
+                {vol.Required(CONF_AUTOMATIC_ADD, default=True): bool}
+            ),
+            errors=errors,
+        )
+
+
+class OptionsFlow(config_entries.OptionsFlow):
+    """Handle flux_led options."""
+
+    def __init__(self, config_entry: config_entries.ConfigEntry) -> None:
+        """Initialize the flux_led options flow."""
+
+        self._config_entry = config_entry
+        self._global_options = None
+        self._device_registry = None
+        self._configure_device = None
+        self._title = "FluxLED/MagicHome"
+
+    async def async_step_init(self, user_input=None):
+        """Manage the options."""
+        return await self.async_step_prompt_options()
+
+    async def async_step_prompt_options(self, user_input=None):
+        """Manage the options."""
+
+        errors = {}
+
+        if user_input is not None:
+            self._global_options = {
+                CONF_AUTOMATIC_ADD: user_input[CONF_AUTOMATIC_ADD],
+                CONF_EFFECT_SPEED: user_input[CONF_EFFECT_SPEED],
+            }
+
+            if CONF_CONFIGURE_DEVICE in user_input:
+                self._configure_device = user_input[CONF_CONFIGURE_DEVICE]
+                return await self.async_step_configure_device()
+
+            if CONF_REMOVE_DEVICE in user_input:
+                device_id = user_input[CONF_REMOVE_DEVICE]
+                del self._config_entry.data[CONF_DEVICES][device_id]
+
+                options_data = self._config_entry.options.copy()
+                if device_id in options_data:
+                    del options_data[device_id]
+                options_data["global"] = self._global_options
+
+                return self.async_create_entry(title="", data=options_data)
+
+            if CONF_HOST in user_input:
+                device_name = (
+                    user_input[CONF_NAME]
+                    if CONF_NAME in user_input
+                    else user_input[CONF_HOST]
+                )
+                device_id = user_input[CONF_HOST].replace(".", "_")
+                device_data = {
+                    CONF_HOST: user_input[CONF_HOST],
+                    CONF_NAME: device_name,
+                }
+                self._config_entry.data[CONF_DEVICES][device_id] = device_data
+
+                options_data = self._config_entry.options.copy()
+                options_data["global"] = self._global_options
+                options_data[device_id] = {CONF_EFFECT_SPEED: DEFAULT_EFFECT_SPEED}
```

Comment:
I don't see where this option is handled in the integration.

Reply:
It was something in the refactored code. I've added this part back into the light.py to handle this part for this PR. Will be updated/changed in the next PR.